### PR TITLE
feat: [a11y docn] Calendar: Update the accessibility documentation

### PR DESCRIPTION
**Pull request by margaree**

### CODE REVIEW

File: components/calendar/README.md [hunk 1]
```diff
@@ -23,17 +23,15 @@ Note: All `*-value` properties should be in ISO 8601 calendar date format (`YYYY
...
-| `summary` | String | Summary of the calendar for accessibility. |
+| `summary` | String | ACCESSIBILITY: Summary of the calendar used by screen reader users for identifying the calendar and/or summarizing its purpose. |
 
 ### Events
 
 * `d2l-calendar-selected`: dispatched when a date is selected through click, space, or enter. `e.detail.date` is in ISO 8601 calendar date format (`YYYY-MM-DD`).
 <!-- docs: end hidden content -->
 
-### Accessibility Properties
+## Accessibility
```
Comment (margaree) [inline components/calendar/README.md:33]: With calendar there was work done that deviated from the w3c recommendation, which is a bit hard to word in the more matter-of-fact bullet point style we've been using for these sections. Let me know suggestions on if we want that to be more consistent.

Comment (dbatiste) [reply components/calendar/README.md:33]: Should we capitalize `w3c`?

File: components/calendar/README.md [hunk 2]
```diff
@@ -23,17 +23,15 @@ Note: All `*-value` properties should be in ISO 8601 calendar date format (`YYYY
...
+The Daylight calendar was built as a part of the date picker initiative and generally follows the w3c's best practice recommendations for a [Date picker dialog](https://www.w3.org/WAI/ARIA/apg/patterns/dialog-modal/examples/datepicker-dialog/). Of note is the keyboard behaviour following the [grid pattern](https://www.w3.org/WAI/ARIA/apg/patterns/grid/).
 
-| Attribute | Description |
-|--|--|
-| `summary` | Use to provide more context on the calendar usage. |
+The w3c recommendations, specifically relating to `attributes` on the calendar's table elements, were followed as much as possible. However, at the time of development it was not possible to create a calendar with the role of `grid` which had the desired appearance, functionality, and supported all browser/screen reader combinations that we support. We did aim to create the same experience for screen reader users as recommended by the example (e.g., announcing the expected content), but had to go about it in a more complex manner. The calendar implementation emphasized the importance of thorough screen reader testing across all supported options.
```
Comment (dbatiste) [inline components/calendar/README.md:37]: Just a couple small tweaks. Feel free to take or leave.
```suggestion
The W3C recommendations, specifically relating to `attributes` on the calendar's table elements, were followed as much as possible. However, at the time of development it was not possible to create a calendar with a `grid` role which had the desired appearance, functionality, and supported all browser/screen reader combinations that we support. We aimed to create the same experience for screen reader users as recommended by the example (e.g., announcing the expected content), but had to go about it in a more complex manner. The calendar implementation emphasized the importance of thorough screen reader testing across all supported options.
```

**Review by dbatiste — APPROVED** [review]
Just a couple of small suggestions. Feel free to update as you see fit. 😄

File: components/calendar/README.md [hunk 3]
```diff
@@ -23,17 +23,15 @@ Note: All `*-value` properties should be in ISO 8601 calendar date format (`YYYY
...
 ### Events
 
 * `d2l-calendar-selected`: dispatched when a date is selected through click, space, or enter. `e.detail.date` is in ISO 8601 calendar date format (`YYYY-MM-DD`).
 <!-- docs: end hidden content -->
 
-### Accessibility Properties
+## Accessibility
 
-To make your usage of `d2l-calendar` accessible, use the following property when applicable:
+The Daylight calendar was built as a part of the date picker initiative and generally follows the w3c's best practice recommendations for a [Date picker dialog](https://www.w3.org/WAI/ARIA/apg/patterns/dialog-modal/examples/datepicker-dialog/). Of note is the keyboard behaviour following the [grid pattern](https://www.w3.org/WAI/ARIA/apg/patterns/grid/).
```
Comment (ChrisLabattD2L) [inline components/calendar/README.md:35]:
```suggestion
`d2l-calendar` was built as part of the date picker initiative and generally follows W3C's best practice recommendations for a [Date picker dialog](https://www.w3.org/WAI/ARIA/apg/patterns/dialog-modal/examples/datepicker-dialog/) with keyboard behaviour that follows the [grid pattern](https://www.w3.org/WAI/ARIA/apg/patterns/grid/)
```

**Review by dbatiste — APPROVED** [review]

File: components/calendar/README.md (hunk 1 above)
Comment (geurts) [reply components/calendar/README.md:33]: Yeah, this does deviate from the matter-of-fact style we use (and not just in bullets) in that it seems to try to explain how the project went, and even calls it the "date picker initiative" which some readers won't have context for (esp. if/when we take the site public) and even a lesson-learned "The calendar implementation emphasized the importance of thorough screen reader testing across all supported options."

I think we need to frame it more simply, without the project context.

**Pull request merged**